labware-library/src/components/App/Page.tsx

```diff
@@ -1,17 +1,16 @@
-// @flow
 import * as React from 'react'
 import cx from 'classnames'
 
 import styles from './styles.css'
 
-export type PageProps = {|
-  scrollRef: { current: HTMLDivElement | null },
-  detailPage: boolean,
-  sidebar: React.Node,
-  content: React.Node,
-|}
+export interface PageProps {
+  scrollRef: { current: HTMLDivElement | null }
```

Comment:
TS has convenience types for the different `ref`s:

```suggestion
  scrollRef: React.RefObject<HTMLDivElement>
```